src/content/docs/develop/resources.mdx

```diff
@@ -34,12 +35,22 @@ Alternatively the `resources` config also accepts a map object if you want to ch
     "resources": {
       "/absolute/path/to/textfile.txt": "resources/textfile.txt",
       "relative/path/to/jsonfile.json": "resources/jsonfile.json",
-      "resources/*": "resources/"
+      "resources/*": "resources/",
+      "resources/**/*": "resources/"
     }
   }
 }
 ```
 
+The syntax for resource source paths works like this:
+
+- `"dir/file.txt"`: Will copy the `file.txt` file into the target resource directory.
+- `"dir/*"` / `dir/**`: Will copy all files in the `dir` directory _non-recursively_ (sub-directories will be ignored) into the target resource directory.
+- `"dir/**/*"`: Will copy all files in the `dir` directory _recursively_ (all files in all sub-directories) into the target resource directory. Note, this does **not** include the files from `dir/*`, but only those in sub-directories.
```

Comment:
Not sure what you mean by does not include files from `dir/*`

Reply:
This is what was confusing me so much in #2708 - this comment was based on 2 things - 1) you saying we're following bash's behavior and 2) the build script fucking with my head.

Since globstar is not enabled by default in bash `*` and `**` behave the same which i didn't notice because i didn't check the config, and didn't look at the output of `**/*` closely enough to notice it missing files 3 levels down.

I'll go through all of it again.

Reply:
It is not that we are following bash's behavior and more like `glob` crate is similar to bash globbing but it is actually more similar to libc's `glob` function